app/internal/page/data.go

```diff
@@ -85,8 +83,21 @@ const (
 	HalfFee FeeType = iota + 1
 	NoFee
 	Hardship
+	Full
```

Comment:
`FullFee` would make more sense when seen out of context

Reply:
and since it is the default you could have it setup like:

```golang
//go:generate enumerator -type FeeType 
type FeeType uint8

const (
  FullFee FeeType = iota
  HalfFee
  NoFee
  Hardship // and maybe change this to HardshipFee too?
)
```

then you wouldn't need to set it in donorStore.Create

Reply:
Ah nice, I hadn't twigged that we get the default when instantiating a new Lpa. And yeah, agree on expanding the const naming.